test/studies/isx/PERFEXECOPTS

```diff
@@ -1 +1 @@
---mode=weakISO --useSubTimers=true
+--mode=weakISO --useSubTimers=true --n 8388608
```

Comment:
I guess since it's only for single locale it's probably ok to keep using the existing problem size. I think we'll want to change it for XC perf testing though.